blocks/adaptors/listoflist_numpy.py

```diff
@@ -10,12 +9,16 @@ class ListOfListsToNumpy(Adaptor):
     inputTypes = DataType.List
     outputType = DataType.NpArray
 
+    def __init__(self, replace_nan: bool) -> None:
+        super().__init__()
+        self.replace_nan = replace_nan
```

Comment:
Could we do this as a config?
Adaptor already subclasses block with has a config on it.

Reply:
I don't mind, but that'd mean I'd need to create a config class and import that whenever I want to use this Adaptor. Does that make sense? Check it out what it'd look like and if you really want me to do it! (I can!)

Reply:
I don't think it's necessary, just probably more standardized across the class system. But I think it might be an overkill now.